templates/gandi/0/rancher-compose.yml

```diff
@@ -0,0 +1,37 @@
+.catalog:
+  name: "Gandi DNS"
+  version: "v0.3.0-rancher1"
+  description: "Rancher External DNS service powered by Gandi. Requires Rancher version 0.44.0"
+  uuid: gandi-0
+  minimum_rancher_version: v0.44.0
+  questions:
+    - variable: "GANDI_APIKEY"
+      label: "Gandi API key"
+      description: "API key Gandi account"
+      type: "string"
+      required: true
+    - variable: "GANDI_TESTING"
+      label: "Testing environment"
+      description: "Whether use the test platform (OT&E)"
+      type: "string"
```

Comment:
@raphink @alena1108 yup, boolean is supported. 
Full list of types supported here: http://docs.rancher.com/rancher/catalog/